src/couch_doc.erl

```diff
@@ -125,7 +125,14 @@ doc_to_json_obj(#doc{id=Id,deleted=Del,body=Body,revs={Start, RevIds},
     }.
 
 from_json_obj({Props}) ->
```

Comment:
Hmm good point. It seems like the cleanest place code-wise as we do other validations in that function (`throw({doc_validation,...`). Wonder if it makes sens to make a validating and non-validation version of the function.